spec/models/merchant_spec.rb

```diff
@@ -140,6 +138,12 @@
   end
 
   describe "instance methods" do
+    describe "#top_five_items" do
+      it "sorts items into top five" do
+        expect(@merchant1.top_five_items).to eq([@item5, @item2, @item4, @item1, @item3])
```

Comment:
We can probably add something similar to above where we demonstrate that the sorting/filtering works.  I'll add that in during refactor.